js/defaults_dialog.js

```diff
@@ -255,36 +335,186 @@ helper.defaultsDialog = (function() {
                 },
                 {
                     key: "failsafe_mission",
-                    value: "OFF"
+                    value: "ON"
                 },
                 {
                     key: "nav_wp_radius",
-                    value: 3000
+                    value: 1500
                 },
+            ],
+            "features":[
                 {
+                    bit: 4, // Enable MOTOR_STOP
+                    state: true
+                }
+            ]
+        },
+		{
+            "title": 'Airplane without a Tail (Wing, Delta, etc)',
+            "notRecommended": false,
+            "id": 3,
+            "reboot": true,
+            "settings": [
+		{
                     key: "platform_type",
                     value: "AIRPLANE"
                 },
-                {
+				{
                     key: "applied_defaults",
                     value: 3
                 },
                 {
+                    key: "gyro_hardware_lpf",
+                    value: "256HZ"
+                },
+                {
+                    key: "gyro_lpf_hz",
+                    value: 25
+                },
+                {
+                    key: "gyro_lpf_type",
+                    value: "BIQUAD"
+                },
+                {
+                    key: "dynamic_gyro_notch_enabled",
+                    value: "ON"
+                },
+                {
+                    key: "dynamic_gyro_notch_q",
+                    value: 250
+                },
+                {
+                    key: "dynamic_gyro_notch_min_hz",
+                    value: 30
+                },
+                {
+                    key: "motor_pwm_protocol",
+                    value: "STANDARD"
+                },
+		{ 
+                    key: "throttle_idle",
+                    value: 5.0
+		},
+                {
+                    key: "rc_yaw_expo",
+                    value: 30
+                },
+                {
+                    key: "rc_expo",
+                    value: 30
+                },
+                {
+                    key: "roll_rate",
+                    value: 18
+                },
+                {
+                    key: "pitch_rate",
+                    value: 9
+                },
+                {
+                    key: "yaw_rate",
+                    value: 3
+                },
+		{ 
+                    key: "nav_fw_pos_z_p",
+                    value: 20
+		},
+		{ 
+                    key: "nav_fw_pos_z_d",
+                    value: 5
+		},
+		{ 
+                    key: "nav_fw_pos_xy_p",
+                    value: 50
+		},
+		{ 
+                    key: "fw_turn_assist_pitch_gain",
+                    value: 0.2
```

Comment:
I only have one plane without a tail, but it needs 0.1 to get smooth turns. In general, I think it is best to err on the low side and lose some altitude that the altitude PIDs will have to correct, then to get this very bumpy pitch behavior during turns when this setting is set too high.

Reply:
@avsaase I tested 0.2 on the Raphax and it was slightly low (dipping nose in turns), it's perfect on the AR Pro and a tad too high on the 250G. I think 0.2 should be a good starting value in general. It also depends on the max navigation bank angle how much gain is needed. so it will be hard to find a perfect value. maybe this should go into the advanced tuning tab in configurator?